From 154c2089204962d290860eeecee58165f72868eb Mon Sep 17 00:00:00 2001
From: Matt DeVillier <matt.devillier@puri.sm>
Date: Tue, 18 May 2021 11:07:16 -0500
Subject: [PATCH 1/4] mb/purism/librem_14: set SA slew rate to fast/16

Decrease SA slew rate to match other domains and reduce
high-frequency noise slightly.

Change-Id: I02cd93481f6bfba6249cb338a0e2f47d471a438e
Signed-off-by: Matt DeVillier <matt.devillier@puri.sm>
---
 .../purism/librem_cnl/variants/librem_14/devicetree.cb          | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/mainboard/purism/librem_cnl/variants/librem_14/devicetree.cb b/src/mainboard/purism/librem_cnl/variants/librem_14/devicetree.cb
index f61eb2b60a..6d0eb205e7 100644
--- a/src/mainboard/purism/librem_cnl/variants/librem_14/devicetree.cb
+++ b/src/mainboard/purism/librem_cnl/variants/librem_14/devicetree.cb
@@ -39,7 +39,7 @@ chip soc/intel/cannonlake
 
 	register "SlowSlewRateForIa" = "3"	# fast/16
 	register "SlowSlewRateForGt" = "3"	# fast/16
-	register "SlowSlewRateForSa" = "2"	# fast/8
+	register "SlowSlewRateForSa" = "3"	# fast/16
 	register "SlowSlewRateForFivr" = "3"	# fast/16
 
 	# Power
-- 
2.30.2

